neo4j/example/src/main/java/example/springdata/neo4j/ActorRepository.java

```diff
@@ -15,11 +15,15 @@
  */
 package example.springdata.neo4j;
 
+import java.util.UUID;
+
 import org.springframework.data.neo4j.repository.GraphRepository;
+import org.springframework.data.neo4j.repository.Neo4jRepository;
 
 /**
  * {@link GraphRepository} for {@link Actor}s.
- * 
+ *
  * @author Luanne Misquitta
+ * @author Mark Angrish
  */
-public interface ActorRepository extends GraphRepository<Actor> {}
+public interface ActorRepository extends Neo4jRepository<Actor, UUID> {}
```

Comment:
Can you clarify on a couple of things here?

- Does the repository actually need to extend a store-specific interface? We usually recommend not to do so as is exposes store details to its clients.
- You change the ID type to `UUID` but the domain class still contains a `Long` identifier? That is at least confusing but actually even looks broken.